website/tests.py

```diff
@@ -145,3 +151,53 @@ def test_on_hide(self):
 
             if "hidden" not in filename:
                 self.assertFalse(True, "files rename failed")
+
+class TestGoogleLogin(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+
+    def test_callback_url(self):
+        view = GoogleLogin()
+        request = self.factory.get("/auth/google/")
+        request.session = {}
+        view.request = request
+
+        def mock_reverse(url_name):
+            if url_name == "google_callback":
+                return "http://example.com/accounts/google/login/callback/"
+            else:
+                raise ValueError("Unexpected URL name")
+
+        def mock_build_absolute_uri(uri):
+            return "http://example.com" + uri
+
+        with patch("django.urls.reverse", side_effect=mock_reverse):
+            with patch("django.http.HttpRequest.build_absolute_uri", side_effect=mock_build_absolute_uri):
+                callback_url = view.callback_url
+
+        self.assertEqual(callback_url, "http://example.com/accounts/google/login/callback/")
```

Comment:
This doesn't seem to be testing anything? What is example.com why are we testing their site?  Also what issue is this solving?

Reply:
How can i directly use BLT domain??

Reply:
Need to figure that out